portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function myFunction(imgs) {
+  // Get the expanded image
+  var expandImg = document.getElementById("expandedImg");
+  // Get the image text
+  var imgText = document.getElementById("imgtext");
+  // Use the same src in the expanded image as the image being clicked on from the grid
+  expandImg.src = imgs.src;
+  // Use the value of the alt attribute of the clickable image as text inside the expanded image
```

Comment:
This is an example of a useful comment, now I know the intention behind the code.

Overall, the code says what operations the program will do.
Comments can clarify, why it is doing those things.